extensions/parquet/table/src/main/java/io/deephaven/parquet/table/pagestore/OffsetIndexBasedColumnChunkPageStore.java

```diff
@@ -56,12 +57,15 @@ private static final class PageState<ATTR extends Any> {
             final long mask,
             @NotNull final ToPage<ATTR, ?> toPage) throws IOException {
         super(pageCache, columnChunkReader, mask, toPage);
-        offsetIndex = columnChunkReader.getOffsetIndex();
+        try (final SeekableChannelContext channelContext =
```

Comment:
We should consider passing in a context. Maybe infeasible. We've deferred the loading, at the cost of not doing it "in bulk" with a single context.

Maybe we can defer loading page stores when we create regions?